### PR TITLE
Merge back release 3.1.8-postman.5

### PR DESCRIPTION
**Pull request by harryi3t**
- [x] published to npm
- [x] published to GitHub release

Release Build: https://buildkite.com/postman/electron-release/builds/42

### CODE REVIEW

File: atom/browser/resources/win/atom.rc [hunk 1]
```diff
@@ -56,8 +56,8 @@ END
 //
 
 VS_VERSION_INFO VERSIONINFO
- FILEVERSION 3,1,8,0
- PRODUCTVERSION 3,1,8,0
```
Comment (kamalaknn) [inline atom/browser/resources/win/atom.rc:60]: @harryi3t Does this signify the point version in the pre-release block?

Comment (harryi3t) [reply atom/browser/resources/win/atom.rc:60]: @kamalaknn  What's a `point version`?
Here, the first 3 numbers are the major, minor & patch version respectively
The last number is gotten by splitting the pre-release section by dot and taking the first number
eg: `x.y.z-someString.a.b.c` => `x,y,z,a`
`x.y.z-someString` => `x,y,z,0`

Comment (kamalaknn) [reply atom/browser/resources/win/atom.rc:60]: That's what I meant. Also why only the first number?

Comment (harryi3t) [reply atom/browser/resources/win/atom.rc:60]: @kamalaknn Windows application only supports 4 digit format. Electron chose to keep the first one as the 4th digit.

**Review by kamalaknn — APPROVED** [review]

**Pull request merged**